popSimManu.tex

```diff
@@ -74,11 +74,9 @@ \section*{Introduction}
 While population genetics has always used statistical methods to make inferences from data,
 the degree of sophistication of the questions, models, data, and computational approaches
 used have all increased over the past decade. Currently there exist myriad computational methods
-that can infer the histories of populations, the distribution of fitness effects,
-recombination rates, the extent of positive selection in genome sequence data
-\citep{li2011inference,schiffels2014inferring,terhorst2017robust,eyre2009estimating,kim2017inference,
-chan2012genome,lin2013fast,Adrion662247,alachiotis2012omegaplus,degiorgio2016sweepfinder2,
-kern2018diplos,sugden2018localization}.
+that can infer the histories of populations \cite{gutenkunst2009inferring,li2011inference,excoffier2013robust,schiffels2014inferring, terhorst2017robust,ragsdale2019models}, the distribution of fitness effects \cite{kim2017inference},
```

Comment:
These should be ``citep`` here to get the right form.

Reply:
I fixed this and cleaned up line breaking as well.